math/gnudatalanguage/Portfile

```diff
@@ -7,7 +7,7 @@ PortGroup                   mpi 1.0
 PortGroup                   github 1.0
 PortGroup                   cxx11 1.1
 
-github.setup                gnudatalanguage gdl 0.9.9 v
+github.setup                gnudatalanguage gdl 1.0.0-rc.1 v
 revision                    3
```

Comment:
When updating the `version`, the `revision` needs to be reset to 0.

Reply:
I think the policy is to only update to stable, released versions; if you want to go for a pre-release versions that should likely be in a ```-devel``` port. Why not wait until  the final v1.0.0 is released upstream?

Reply:
The 0.9.9 is from Nov 2018, I assume we will have good few subsequent 1.0.0 release candidates, and actually it was our intention to gather feedback from users and packagers before shipping a 1.0.0